src/Twig/Extension/StringExtension.php

```diff
@@ -29,18 +29,28 @@
  * @internal
  *
  * @see https://github.com/symfony/symfony/pull/35649
+ * @deprecated since sonata-project/admin-bundle 3.69, to be removed in 4.0.
```

Comment:
You should usually use `sonata-project/admin-bundle 3.x` because you can't be sure that he release will not be created before this merge.

Reply:
I know, but https://github.com/sonata-project/SonataAdminBundle/pull/6145/files/733c89a78260f6cfa810a2d9d8d34bb2564a7383#diff-b96bf7cefcf2cc23077357469e7d42cbR23 was add in 3.69, so in this case it should be OK.